src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -47,6 +47,12 @@ class SensorFusion
      */
     std::optional<World> getWorld() const;
 
+    /**
+     * Overrides the referee commands and updates the game state accordingly if
+     * an override exists.
+     **/
+    void overrideRefereeCommand();
```

Comment:
I think we'd expect that if the dynamic parameters were overridden that there'd be some effect, so it doesn't make sense to me for this function to be exposed like this. I think it should happen automatically

Reply:
discussed and fixed